### DISCUSSION TITLE
gesp-1-bcqm3001-3004/

### DISCUSSION THREAD

**Discussion by giscus** [lihongzheshuai/lihongzheshuai.github.io#338] [General]
# gesp-1-bcqm3001-3004/

四道启蒙基础题，练习基本输出语句cout。

https://www.coderli.com/gesp-1-bcqm3001-3004/

**Comment by Niuger** [comment]
2024.9.27 打卡
问一下这个cout在纯c语言程序里面怎么用呀
还有这个代码为什么在dev里面能跑在vscode就跑不了呀后缀名也是cpp
